x/docs/md/cli/completion.md

```diff
@@ -0,0 +1,46 @@
+### Setting up Exercism CLI completion (optional)
+
+#### Bash
+
+If you use a Bash shell, you can use the CLI Bash-completion script.
+
+First download the script [[view source]](http://cli.exercism.io/exercism_completion.bash):
+
+```bash
+mkdir -p ~/.config/exercism/
+curl http://cli.exercism.io/exercism_completion.bash > ~/.config/exercism/exercism_completion.bash
+```
+
+Load up the completion in your `.bashrc`, `.bash_profile` or `.profile` by adding the following snippet:
+
+```bash
+if [ -f ~/.config/exercism/exercism_completion.bash ]; then
+  . ~/.config/exercism/exercism_completion.bash
+fi
+```
+
+After opening a new Bash shell, you should be able to type `exercism s` and Bash completion will give you `exercism submit`.
+
+#### Zsh
+
+If you use a Zsh shell, you can use the CLI Zsh-completion script.
+
+First download the script [[view source]](http://cli.exercism.io/exercism_completion.zsh):
+
+```zsh
+mkdir -p ~/.config/exercism/
+curl http://cli.exercism.io/exercism_completion.bash > ~/.config/exercism/exercism_completion.bash
```

Comment:
This is a typo: it should be *.zsh not *.bash for the curl; it won't work otherwise.

Thought this would be the right place to point it out as such it's a small change - but it's my first ever contribution so please correct me if I'm wrong!